### ISSUE TITLE
Add CHANGELOG.md

### ISSUE THREAD

**Issue by rytisder** [amirkhiz/resellerclub-php-api#22]
Standard: https://keepachangelog.com/en/1.0.0/

Example: https://github.com/guzzle/guzzle/blob/master/CHANGELOG.md

**Comment by amirkhiz** [comment]
@rytisder Good idea. I have tried to add a minified changelog for every release note but this will be better to have it as a separate file in the repository. thanks :)
